app/internal/event.py

```diff
@@ -65,22 +73,71 @@ def check_diffs(checked_event: Event,
 
 
 def find_pattern(session, event):
-    all_events_with_same_name = session.query(Event).filter(
-        Event.owner_id == event.owner_id, Event.title == event.title).all()
+    all_events_with_same_name = (
+        session.query(Event)
+        .filter(Event.owner_id == event.owner_id, Event.title == event.title)
+        .all()
+    )
 
     return check_diffs(event, all_events_with_same_name)
 
 
-def get_messages(session: Session,
-                 event: Event,
-                 uninvited_contacts: Set[str]) -> List[str]:
+def get_messages(
+    session: Session,
+    event: Event,
+    uninvited_contacts: Set[str],
+) -> List[str]:
     messages = []
     if uninvited_contacts:
-        messages.append(f'Forgot to invite '
-                        f'{", ".join(uninvited_contacts)} maybe?')
+        messages.append(
+            f"Forgot to invite " f'{", ".join(uninvited_contacts)} maybe?',
+        )
 
     pattern = find_pattern(session, event)
     for weeks_diff in pattern:
-        messages.append(f'Same event happened {weeks_diff} weeks before too. '
-                        f'Want to create another one {weeks_diff} after too?')
+        messages.append(
+            f"Same event happened {weeks_diff} weeks before too. "
+            f"Want to create another one {weeks_diff} after too?",
+        )
     return messages
+
+
+def add_countries_to_db(session: Session) -> None:
+    """
+    Adding all new countries to the "Country" table in the database.
+    Information is based on the "countries" list.
+    (The list is located in app/resources/countries.py)
+    Names are described either as:
+    "Country Name, City Name" or
+    "Country Name" solely.
+    Timezones are described as "Continent/ City Name"
+    for example:
+        name: Israel, Jerusalem
+        timezone: Asia/Jerusalem
+    """
+    for country in countries:
+        partial_name = country["name"]
+        for capital_city in country["timezones"]:
+            capital_city_name = capital_city.split("/")[-1]
+            if partial_name != capital_city_name:
+                name = partial_name + ", " + capital_city_name
+            else:
+                name = capital_city_name
+            existing = session.query(Country).filter_by(name=name).first()
+            if not existing:
+                new_country = Country(name=name, timezone=str(capital_city))
+                session.merge(new_country)
+    session.commit()
+    session.close()
+
+
+@functools.lru_cache
+def get_all_countries_names(session: Session) -> List:
```

Comment:
`List[...]`?

Reply:
You probably marked as resolved but forget to push the changes?